src/main/markdown/makinggwtbetter.md

```diff
@@ -614,9 +617,7 @@ mvn jetty:run
 and point your browser to [http://localhost:8080](http://localhost:8080).
 
 If you think your change is ready to be published on gwtproject.org you can send us
-your changes for review by using gerrit. Note: you need to have signed a <a href="#clas">CLA Agreement</a>.
-
-This works the same way you would submit a code change to GWT. More information can be found [here](#submittingpatches).
+your changes with a pull request to http://github.com/gwtproject/gwt-site.
```

Comment:
No need the googlebot will post instructions on the pull request
On Jan 10, 2015 3:59 PM, "Julien Dramaix" <notifications@github.com> wrote:

> In src/main/markdown/makinggwtbetter.md
> <https://github.com/gwtproject/gwt-site/pull/9#discussion-diff-22759441>:
>
> > @@ -614,9 +617,7 @@ mvn jetty:run
> >  and point your browser to [http://localhost:8080](http://localhost:8080).
> >
> >  If you think your change is ready to be published on gwtproject.org you can send us
> > -your changes for review by using gerrit. Note: you need to have signed a <a href="#clas">CLA Agreement</a>.
> > -
> > -This works the same way you would submit a code change to GWT. More information can be found [here](#submittingpatches).
> > +your changes with a pull request to http://github.com/gwtproject/gwt-site.
>
> Shouldn't we keep the note about the cla somewhere ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gwtproject/gwt-site/pull/9/files#r22759441>.
>